TLM/TLM/UI/ModUI.cs

```diff
@@ -11,17 +11,8 @@ namespace TrafficManager.UI {
     /// Access via ThreadingExtension.ModUi.
     /// </summary>
     public class ModUI : UICustomControl {
-        /// <summary>Singleton storage.</summary>
-        public static ModUI instance_;
-
         /// <summary>Singleton accessor.</summary>
-        public static ModUI Instance {
-            get => instance_;
-        }
-
-        public static void SetSingletonInstance(ModUI newInstance) {
-            instance_ = newInstance;
-        }
+        public static ModUI Instance { get; private set; }
```

Comment:
Can this not be simplified to just `public static ModUI Instance { get; }` ?

Reply:
This would only work if we can assign to Instance in the Constructor.
Instance is set in OnLevelLoaded and unset in Destroy though, so we need the private setter :/.
This could be done cleaner, but I only wanted to do the minimal changes to accomodate Finalizers removal.